app/services/proofing/aamva/applicant.rb

```diff
@@ -11,6 +11,10 @@ def self.from_proofer_applicant(applicant)
           last_name: applicant[:last_name],
           dob: format_dob(applicant[:dob]),
           state_id_data: format_state_id_data(applicant),
+          address1: applicant[:address1],
```

Comment:
is there an address2?

Reply:
There is an address2 though I am just throwing the first line in for now since getting the 2nd in is a little complicated.

Reply:
To elaborate on this, the way the docs say to add the address to the request is to include an additional `AddressDeliveryPointText` field for address line 2, but _only_ if there is an address line 2 present.

This means adding a little bit of extra logic to this class to handle that. We probably should do that but I'm keeping it simple for now to verify everything is working then we can come back around and handle the second address line.